string rosBagFile
string[] topicsToRecord
bool useDefaultTopics
bool startRecording
---
bool recordingActive
string recordingStatus
